docsrc/conf.py

```diff
@@ -13,8 +13,17 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.abspath("../../src"))
-
+sys.path += [
+	os.path.abspath("../src"), 
+	os.path.abspath("../src/firebolt/"), 
+	os.path.abspath("../src/firebolt/async_db"), 
+	os.path.abspath("../src/firebolt/client"), 
+	os.path.abspath("../src/firebolt/common"),
+	os.path.abspath("../src/firebolt/db"),
+	os.path.abspath("../src/firebolt/model"), 
+	os.path.abspath("../src/firebolt/service")
+	]
```

Comment:
This may be brittle if/when we refactor modules. For example, I plan to restructure modules once some of the current PRs are done: https://packboard.atlassian.net/browse/FIR-9390

Why do we need `"../src"` and also subpaths like `"../src/firebolt/"` etc?

Reply:
The "../src/firebolt/" line can probably be removed, but unfortunately listing out each and every module folder was the only way ReadtheDocs would import everything